### DISCUSSION TITLE
New hosting provider free

### DISCUSSION THREAD

**Discussion by MicroChipKali** [3kh0/ext-remover#689] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My proxy hosting provider got blocked so i need new ones before school starts.

**Comment by Aka-but-nice** [comment]
Cope.

**Comment by obkduishinijf** [comment]
how about working instead of playing coolmathgames 24/7

**Comment by cinnamongold** [comment, marked as answer]
if its rly that serious just use cosmicgeometry.xyz

or just do your damn work
